engine/player/soulbinds.cpp

```diff
@@ -453,18 +453,33 @@ void thrill_seeker( special_effect_t& effect )
     make_repeating_event( *p->sim, eff_data->period(), [ counter_buff ] { counter_buff->trigger(); } );
   } );
 
-  auto p                  = effect.player;
-  int killing_blow_stacks = as<int>( p->find_spell( 331586 )->effectN( 1 ).base_value() );
+  auto p                     = effect.player;
+  int killing_blow_stacks    = as<int>( p->find_spell( 331586 )->effectN( 1 ).base_value() );
+  double killing_blow_chance = p->sim->shadowlands_opts.thrill_seeker_killing_blow_chance;
+  // If the user does not override the value for this we will set different defaults based on the sim here
+  // Default: 1/20 = 0.05
```

Comment:
why not give the values 4 and 20 a name? num_players or something. And then just set killing_blow_chance = 1.0 / num_players;
Makes your code self-documenting and easier to understand.

Also why 1/4 for dungeon slice instead of 1/5, but still 1/20 for a raid?

Reply:
I think in both of these cases the reasons are because they arent directly tied to players. Dungeons should have a significantly higher chance, I just chose 4 as a conservative default. As for the 1/20 I think the same applies there, don't want to assume mythic raid, could easily have 20 dps with a 30 man heroic roster or something.

The comment here is probably doing more harm than good in this case. I think the better way to just look at it is the percents rather than how we randomly decided on these two figures :)